test/cloud-event.ts

```diff
@@ -89,21 +80,15 @@ describe("parseCloudEvent", () => {
       sffncontext: Buffer.from("{", "utf-8").toString("base64"),
     });
     const { headers, body } = HTTP.binary(inputEvent);
-    const { cloudEvent, sfContext, sfFunctionContext } = parseCloudEvent(
-      headers,
-      body
-    );
 
-    expect(cloudEvent.id).to.equal(inputEvent.id);
-    // Should this even be `null`, or something else?
-    expect(sfContext).to.equal(null);
-    expect(sfFunctionContext).to.equal(null);
+    try {
+      parseCloudEvent(headers, body);
+    } catch (error) {
+      expect(error.toString()).to.include("Unexpected end of JSON input");
```

Comment:
I think the new error messaging you added is more important to check for. Maybe add an assertion about it?
```suggestion
      expect(error.toString()).to.include("Function arguments could not be determined due to invalid JSON");
      expect(error.toString()).to.include("Unexpected end of JSON input");
```